atlasdb-ete-test-utils/src/main/java/com/palantir/atlasdb/ete/Gradle.java

```diff
@@ -42,10 +42,10 @@ public static Gradle ensureTaskHasRun(String command) {
     @Override
     protected void before() throws Throwable {
         if (isRunningOutsideOfGradle()) {
-            System.out.println("It looks like you are not running in gradle, performing the required gradle command: " + command);
+            System.out.println("It looks like you are not running in gradle, performing the required gradle command: " + command);  // (authorized)
```

Comment:
this is super old, but was in my queue, I believe this comes from a checkstyle we have (at least internally) that prevents any system.out.printlns that don't have an authorized block.